ads/google/a4a/google-data-reporter.js

```diff
@@ -140,16 +140,16 @@ export function getLifecycleReporter(ampElement, namespace, slotId) {
  * non-Google networks, returns a BaseLifecycleReporter -- a stub reporter that
  * generates no outputs.
  *
- * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4aElement
+ * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A|!../../../extensions/amp-ad/0.1/amp-ad-3p-impl.AmpAd3PImpl} element
  * @param {string=} opt_namespace  CSI ping namespace.  For example, a key
  *   of #ReporterNamespace.
  * @return {!./performance.BaseLifecycleReporter}
  */
-export function googleLifecycleReporterFactory(a4aElement, opt_namespace) {
+export function googleLifecycleReporterFactory(element, opt_namespace) {
```

Comment:
Oops.  Hadn't noticed the `element.element` when I did the auto-rename.  `baseInstance` sounds good.  Fixed.